srcpkgs/sbt/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'sbt'
+pkgname=sbt
+version=1.2.3
+revision=1
+noarch=yes
+wrksrc=${pkgname}
+hostmakedepends="wget"
+depends="virtual?java-environment"
+short_desc="The interactive build tool for Scala and Java"
+maintainer="Damian Czaja <trojan295@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://www.scala-sbt.org"
+distfiles="https://sbt-downloads.cdnedge.bluemix.net/releases/v${version}/${pkgname}-${version}.tgz"
+checksum="d502fbe587a6c2181d6acc688741ae4131386bb10ca50c73c923effc60bafeeb"
+
+do_install() {
+    vmkdir usr/share/sbt
+    vmkdir usr/bin
+    vcopy "*" usr/share/sbt
+    ln -sf /usr/share/sbt/bin/sbt ${DESTDIR}/usr/bin/sbt
+    wget https://raw.githubusercontent.com/sbt/sbt/v${version}/LICENSE
```

Comment:
no need, put the URL in the distfiles with `>LICENSE.txt`

then do `vlicense ../LICENSE.txt`